docs/usage/CATEGORY_SYNC.md

```diff
@@ -47,46 +47,16 @@ matched.
     - Retries on 5xx errors with a retry strategy. This can be achieved by decorating the `sphereClient` with the 
    [RetrySphereClientDecorator](http://commercetools.github.io/commercetools-jvm-sdk/apidocs/io/sphere/sdk/client/RetrySphereClientDecorator.html)
    
-  If you have no special requirements on sphere client creation then you can use `ClientConfigurationUtils#createClient`
-  util which applies best practices already.
+   If you have no special requirements on sphere client creation then you can use `ClientConfigurationUtils#createClient` 
+   util which applies best practices already.
```

Comment:
```suggestion
   util, which applies best practices already.
```

Reply:
I don't this comma is grammatically correct: https://en.oxforddictionaries.com/punctuation/comma